lib/drivers/tandemTslimDriver.js

```diff
@@ -1650,6 +1685,76 @@ module.exports = function (config) {
     return records;
   };
 
+  var buildAlarmRecords = function (data,records) {
+    var alarmRecords = filterLogEntries([PUMP_LOG_RECORDS.LID_ALARM_ACTIVATED,PUMP_LOG_RECORDS.LID_ALERT_ACTIVATED],data.log_records);
+    alarmRecords.forEach(function (alarmEntry) {
+      var alarmRecord = cfg.builder.makeDeviceEventAlarm()
+        .with_deviceTime(alarmEntry.deviceTime)
+        .set('index', alarmEntry.log_index);
+      cfg.tzoUtil.fillInUTCInfo(alarmRecord, alarmEntry.jsDate);
```

Comment:
All the alarms (now that I added a call to the `buildAlarmRecords` fn locally) have an `uncertain-timestamp` annotation. This means the bootstrapping module is confused, so check through everything here.